content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+
+
+### Step 1: Assume people will be receptive
+
+Everyone started where you are (i.e. with no network and no knowledge) and many of them are open to having conversations either out of genuine friendliness or a desire to "pay it forward." When you first start it may be scary to reach out to people with whom you have a tenuous connection at best, but remember people are generally nice. You will be surprised who will respond (for example, [a twelve year old Steve Jobs successfully reached out to Bill Hewlett, the CEO of HP](https://www.businessinsider.com/what-we-can-learn-from-the-gutsy-way-steve-jobs-landed-a-job-at-hp-2015-7)). After all, what is the worst that can happen - they say no or don't respond at all?
+
+
+### Step 2: Set a goal and do your research
+
+Make sure you have a defined knowledge goal before you begin reaching out to people and be as specific as possible. There is nothing worse than giving your time to talk to someone when they don't know what they want. 
+
+In my case my goal was: 
+
+>I am building an Analytics team, so I am looking to speak with others who have done it before. Specifically I would like to learn about specialist vs generalist models, titles and how the Analytics team interfaces with the rest of the organization.
+
+I also recommend doing specific research on the people you contact to understand what relevant experience or knowledge they may have. This will allow you to craft your message and, in the event of a conversation, your questions.
+
+
+### Step 3: Reach out to your warmest connections (and be accommodating)
+
+Message your warmest connections who may have relevant knowledge or may know others who have relevant knowledge. If you don't have any warm connections, contact people who you have something in common with (e.g., university, home town, industry). Linkedin is a great place to meet new contacts, but industry-specific or topic-specific messaging boards / Slack communities are even better.
+
+Craft your message so that it is specific, friendly and most importantly accommodating. Make meeting you sound like a very light lift. Here is a message I sent many years ago:
+
+>Hi XXX, I am building out an Analytics team and would love to chat with you to learn about how you've gone about building your team at YYY company. I can imagine you have similar challenges (e.g., interfacing with stakeholders and deciding on the generalist vs specialist model). Happy to work around your schedule and come to you. Let me know when works best.
+
+
+### Step 4: Ask good questions and make a genuine connection
+
+People can quickly tell if you are being disingenuous and only want a job or an introduction. I advise going into the conversation with the goal of learning and potentially forming a professional or personal relationship - a job or an introduction is icing on the cake. I met many of my closest friends in the NYC scene through conversations just like this.
```

Comment:
I think this is important -- you could tease this out a bit more, I think.

I'd say that it's incredibly important not to mislead people ... they're donating time to you, and you should be genuine in your goals. If you're looking for a job or a favor or something else, that's a very different conversation than what I'm describing here, and you'll want to take a different approach ...

^^ or something like that